### PR TITLE
adding works

### PR DESCRIPTION
**Pull request by gyrydymptum**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: max-of-vector
🏅 Granted 5 XP for @gyrydymptum

**Comment by ziobron** [comment]
⏱ 3 XP extra for @gyrydymptum for on-time delivery

File: homework/max-of-vector/maxOfVector.hpp [hunk 1]
```diff
@@ -3,6 +3,10 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+int max = vec[0];
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:6]: Wrong indentation.

Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:6]: What if vec is empty?
Mark max as const

File: homework/max-of-vector/maxOfVector.hpp [hunk 2]
```diff
@@ -3,6 +3,10 @@
...
+    for (const auto& el :vec) {
+        if (max < el) 
+        max = el;
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:9]: Wrong indentation.

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**